src/test/files/architecture/architectureDevonProject.json

```diff
@@ -0,0 +1,10 @@
+{
+  "architecture": {
+    "devonProject": true,
```

Comment:
This property seems pointless to me.
If we do not override `packages` config, we are using the defaults for the package-structure defined by `devon4j` coding-conventions.

Reply:
ok, you're right, I will remove this.